public/components/createPreviewWindow.js

```diff
@@ -9,6 +9,9 @@ const windowParameters = {
   show: false,
   width: 1024,
   height: 768,
+  webPreferences: {
+    nodeIntegration: true,
```

Comment:
As per my comment in the Network Canvas PR, it would be nice to be able to lock this down.